vite.dspublisher.ts

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+import type { UserConfig } from 'vite';
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
nit: let's add a newline after `import` block:

```suggestion

const __filename = fileURLToPath(import.meta.url);
```